v4/export/dump.go

```diff
@@ -170,6 +170,11 @@ func Dump(pCtx context.Context, conf *Config) (err error) {
 
 	if conf.Consistency != "lock" {
 		conn := connectPool.getConn()
+		// record again, to provide a location to exit safe mode for DM
+		err = m.recordGlobalMetaData(conn, conf.ServerInfo.ServerType, true)
```

Comment:
How about only recording this for consistency `none`?

Reply:
What's more, I think tidb also doesn't need this.

Reply:
how about expose a variable, and only DM using embebded dumpling turns on this feature

Reply:
Sounds nice. I think add this feature to `export.Conf` is OK. We don't have to expose it to dumpling control.

Reply:
https://github.com/pingcap/dumpling/pull/146/commits/dae2f0bbda9dc06f0190c61ba3bddf59a9d6246e